src/app/hooks/use-balances-loading.ts

```diff
@@ -49,14 +52,16 @@ export const useBalancesLoading = () => {
             dispatch(putTokensBalancesAction({ publicKeyHash, chainId, balances }));
           }
           break;
+        default:
```

Comment:
I bumped into the problem that this message wasn't sent in case of reconnection because of a page refresh. Please try this scenario, and use `connectionReady` for preventing requests on each block instead if it is still not sent.